backend/btrixcloud/colls.py

```diff
@@ -106,83 +178,49 @@ async def list_collections(
 
         total = await self.collections.count_documents(match_query)
 
-        cursor = self.collections.find(
-            match_query, projection=["_id", "name"], skip=skip, limit=page_size
-        )
+        cursor = self.collections.find(match_query, skip=skip, limit=page_size)
         results = await cursor.to_list(length=page_size)
-        collections = [CollOut.from_dict(res) for res in results]
+        collections = [Collection.from_dict(res) for res in results]
 
         return collections, total
 
-    async def get_collection_crawls(self, oid: uuid.UUID, name: str = None):
-        """find collection and get all crawls by collection name per org"""
-        collid = None
-        if name:
-            coll = await self.find_collection(oid, name)
-            if not coll:
-                return None
-
-            collid = coll.id
+    async def get_collection_crawls(self, oid: uuid.UUID, name: str):
+        """Find collection and get all crawls by collection name per org"""
 
-        crawls = await self.crawls.list_finished_crawls(oid=oid, collid=collid)
-        all_files = []
-        for crawl in crawls:
-            if not crawl.files:
-                continue
+        coll = await self.get_collection(oid, name)
+        if not coll:
+            raise HTTPException(status_code=404, detail="collection_not_found")
 
-            for file_ in crawl.files:
-                if file_.def_storage_name:
-                    storage_prefix = (
-                        await self.crawl_manager.get_default_storage_access_endpoint(
-                            file_.def_storage_name
-                        )
-                    )
-                    file_.filename = storage_prefix + file_.filename
+        collection_crawls = {}
 
-                all_files.append(file_.dict(exclude={"def_storage_name"}))
+        for crawl_id in coll.crawl_ids:
+            org = await self.orgs.get_org_by_id(oid)
+            crawl = await self.crawls.get_crawl(crawl_id, org)
+            collection_crawls[crawl_id] = crawl.resources
```

Comment:
We actually want to return a flat array of `resources` objects here, so that this can conform with the 'MultiWACZ' Frictionless Data-based spec, where each resource is a WACZ file from the crawl.
It looks like frictionless requires the resources list to be flat: https://specs.frictionlessdata.io/data-package/#resource-information
Depending on how the spec evolves, the result could be:
```
[
 {"name": "<crawl id>/<filename.wacz>
  "path": "<presigned url>
 }
 ]
 ```
 or maybe:
 ```
 [
 {
  "crawl": "<crawl id>"
  "name": "<filename.wacz>"
  "path": "<presigned url>
  }]
 ```

Reply:
Or, we keep this as is, and add a separate `/collections/<name>/replay.json` endpoint, like we do for crawls?

Reply:
Ah okay, that makes sense! I was missing the context for this. Happy to make this a flat array - I don't know that we have a use case for this endpoint as a separate entity to warrant a separate replay.json endpoint?

Reply:
Modified so that the result looks like:

```
{"resources":
[
  {
      "name":"84e35263-3bfd-4248-9b75-baf49c8e73ff/20230413163255245-manual-20230413163208-abb695ed-1a4.wacz",
      "path":"/data/84e35263-3bfd-4248-9b75-baf49c8e73ff/20230413163255245-manual-20230413163208-abb695ed-1a4.wacz?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=ADMIN%2F20230413%2F%2Fs3%2Faws4_request&X-Amz-Date=20230413T190612Z&X-Amz-Expires=3600&X-Amz-SignedHeaders=host&X-Amz-Signature=6b474aa09c37baac6a7b56a8c7e97371d68cbd27dd8b640d83b7489add4c1de3","hash":"637feafabe5bef0dbafcfd2b2e8dae8b07d3be9a85b966879f504758123c465a",
      "size":174677,
      "crawlId":"manual-20230413163208-abb695ed-1a4"
  }
]}
```

Reply:
Nice, that works! Yeah, can revisit separate endpoint if needed for some other reason